### PR TITLE
Document how to publish updated documentation

### PR DESCRIPTION
**Pull request by raboof**

### CODE REVIEW

**Review by ennru — APPROVED** [review]
LGTM.

File: RELEASING.md [hunk 1]
```diff
@@ -60,6 +60,27 @@ copied over to a maven server. If you have access, the Jenkins job at
 https://jenkins.akka.io:8498/job/akka-publish-wip/ can be used to publish
 a snapshot to https://repo.akka.io/snapshots from any branch.
 
+## Releasing only updated docs
+
+It is possible to release a revised documentation to the already existing release.
+
+1. Create a new branch from a release tag. If a revised documentation is for the `v2.6.4` release, then the name of the new branch should be `docs/v2.6.4`.
+1. Add and commit `version.sbt` file that pins the version to the one, that is being revised. Also set `isSnapshot` to `false` for the stable documentation links. For example:
```
Comment (renatocaval) [inline RELEASING.md:68]:
```suggestion
1. Add and commit `version.sbt` file that pins the version to the one that is being revised. Also set `isSnapshot` to `false` for the stable documentation links. For example:
```

**Review by octonato — APPROVED** [review]

**Review by octonato — APPROVED** [review]
One minor change request. LGTM otherwise.

**Review by johanandren — APPROVED** [review]

**Review by patriknw — COMMENTED** [review]
great to have that written down

**Pull request merged**